{"timestamp":"2025-06-20T15:07:24.865609","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-06-20T15:07:24.867599","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/etl_proj.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-06-20T15:07:25.557994","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"ProgrammingError","exc_value":"(psycopg2.errors.InvalidSchemaName) schema \"data\" does not exist\nLINE 2: CREATE TABLE data.recalls (\n                     ^\n\n[SQL: \nCREATE TABLE data.recalls (\n\trecall_id BIGINT, \n\ttitle TEXT, \n\torganization TEXT, \n\tproduct_name TEXT, \n\tissue TEXT, \n\tcategory TEXT, \n\tupdated_at TEXT, \n\tdata_received_at TEXT, \n\tissue_category TEXT, \n\tdata_ingested_at TIMESTAMP WITHOUT TIME ZONE\n)\n\n]\n(Background on this error at: https://sqlalche.me/e/14/f405)","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/etl_proj.py","lineno":83,"name":"load_to_db"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/util/_decorators.py","lineno":333,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/core/generic.py","lineno":3008,"name":"to_sql"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/sql.py","lineno":788,"name":"to_sql"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/sql.py","lineno":1948,"name":"to_sql"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/sql.py","lineno":1852,"name":"prep_table"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/sql.py","lineno":936,"name":"create"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/pandas/io/sql.py","lineno":922,"name":"_execute_create"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/sql/schema.py","lineno":962,"name":"create"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":2221,"name":"_run_ddl_visitor"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/sql/visitors.py","lineno":524,"name":"traverse_single"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/sql/ddl.py","lineno":899,"name":"visit_table"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":1385,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/sql/ddl.py","lineno":80,"name":"_execute_on_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":1477,"name":"_execute_ddl"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":1953,"name":"_execute_context"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":2134,"name":"_handle_dbapi_exception"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/util/compat.py","lineno":211,"name":"raise_"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":1910,"name":"_execute_context"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/default.py","lineno":736,"name":"do_execute"}],"is_group":false,"exceptions":[]},{"exc_type":"InvalidSchemaName","exc_value":"schema \"data\" does not exist\nLINE 2: CREATE TABLE data.recalls (\n                     ^\n","exc_notes":[],"syntax_error":null,"is_cause":true,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/base.py","lineno":1910,"name":"_execute_context"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/sqlalchemy/engine/default.py","lineno":736,"name":"do_execute"}],"is_group":false,"exceptions":[]}]}
